build-latest.sh

```diff
@@ -15,12 +15,12 @@ fi
 # Checking for the presence of GITHUB_OAUTH_CLIENT_ID
 # and GITHUB_OAUTH_CLIENT_SECRET
 ###
-if [ -n "${GITHUB_OAUTH_CLIENT_ID}" ] && [ -n "${GITHUB_OAUTH_CLIENT_SECRET}" ]; then
+if [ -n "${GITHUB_TOKEN}" ]; then
   echo "🗝 Performing authenticated Github API calls."
-  GITHUB_OAUTH_PARAMS="client_id=${GITHUB_OAUTH_CLIENT_ID}&client_secret=${GITHUB_OAUTH_CLIENT_SECRET}"
+  CURL_AUTH_HEADER="--header 'authorization: Bearer ${GITHUB_TOKEN}'"
```

Comment:
I believe the 'A' of Authorization is usually capitalized in HTTP headers. But it probably doesn't matter and curl does it for you.